betalife-app/src/components/Intro.js

```diff
@@ -0,0 +1,311 @@
+import React from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { MDBNavbar, MDBNavbarBrand, MDBNavbarNav, MDBNavbarToggler, MDBCollapse, MDBNavItem, MDBNavLink, MDBContainer, MDBMask, MDBView, MDBIcon, MDBBtn, MDBModal, MDBModalBody, MDBModalHeader, MDBInput, MDBFormInline} from "mdbreact";
+import { BrowserRouter as Router } from "react-router-dom";
+import loginBg from "../images/buildings.jpg";
+
+class Intro extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      collapse: false,
+      isWideEnough: false,
+      modal2: false,
+      modal3: false,
+      modal4: false,
+      radio: 2,
+      startDate: new Date()
+    };
+    this.doCollapse = this.doCollapse.bind(this);
+  }
+
+  doCollapse() {
+    this.setState({
+      collapse: !this.state.collapse
+    });
+  }
+
+  toggle = (nr) => () => {
+    if (nr === 2) {
```

Comment:
What does nr represent please?

Reply:
Nice observation. I'll be adding comments going forward.